routers/api/v1/user/key.go

```diff
@@ -198,6 +199,11 @@ func GetPublicKey(ctx *context.APIContext) {
 
 // CreateUserPublicKey creates new public key to given user by ID.
 func CreateUserPublicKey(ctx *context.APIContext, form api.CreateKeyOption, uid int64) {
+	if setting.Admin.UserDisabledFeatures.Contains(setting.UserFeatureSSHKeys) {
```

Comment:
Sounds like SSH would be disabled completely.
I wonder, should we rename the keys to make it more obvious that only the configuration is forbidden?

Reply:
I think in context it probably makes enough sense, and presumably an admin would have to read the docs to find the key in the first place. 
But, an alternative name could be `manage_ssh_keys` or similar?

Reply:
Done.